Framework/Shimmer.h

```diff
@@ -0,0 +1,19 @@
+/**
+ Copyright (c) 2014-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+#import <UIKit/UIKit.h>
+
+//! Project version number for Shimmer.
+FOUNDATION_EXPORT double ShimmerVersionNumber;
+
+//! Project version string for Shimmer.
+FOUNDATION_EXPORT const unsigned char ShimmerVersionString[];
+
```

Comment:
Ah, maybe this is paired with the versioning support file Xcode also generates. Either way, I don't see a good reason for it, so I'd be fine with disabling the versioning support in Xcode and removing it here.